README.md

```diff
@@ -64,7 +74,7 @@ We also have official client-side implementations for
 [Java](https://github.com/metarhia/jstp-java)
 that work effortlessly on iOS and Android 🎉
 
-## Getting Started
+<h2 align="center">Getting Started</h2>
```

Comment:
@aqrln, as I said in https://github.com/metarhia/jstp/pull/274#discussion_r129813296, I don't like the style changes, can we discuss the other possible alternatives?